inject.js

```diff
@@ -204,158 +274,159 @@ chrome.runtime.sendMessage({}, function(response) {
-              || event.getModifierState("Control")
-              || event.getModifierState("Fn")
-              || event.getModifierState("Meta")
-              || event.getModifierState("Hyper")
-              || event.getModifierState("OS")) {
-            return;
-          }
+    if (document === window.document) {
+      defineVideoController();
+    } else {
+      var link = document.createElement('link');
+      link.href = chrome.runtime.getURL('inject.css');
+      link.type = 'text/css';
+      link.rel = 'stylesheet';
+      document.head.appendChild(link);
+    }
+    var docs = Array(document)
+    try {
+      if (inIframe())
+        docs.push(window.top.document);
+    } catch (e) {
+    }
+    docs.forEach(function (doc) {
+      doc.addEventListener('keydown', function (event) {
+        var keyCode = event.keyCode;
+
+        // Ignore if following modifier is active.
+        if (!event.getModifierState
+          || event.getModifierState("Alt")
+          || event.getModifierState("Control")
+          || event.getModifierState("Fn")
+          || event.getModifierState("Meta")
+          || event.getModifierState("Hyper")
+          || event.getModifierState("OS")) {
+          return;
+        }
 
-          // Ignore keydown event if typing in an input box
-          if ((document.activeElement.nodeName === 'INPUT'
-                && document.activeElement.getAttribute('type') === 'text')
-              || document.activeElement.nodeName === 'TEXTAREA'
-              || document.activeElement.isContentEditable) {
-            return false;
-          }
+        // Ignore keydown event if typing in an input box
+        if (document.activeElement.nodeName === 'INPUT'
+            || document.activeElement.nodeName === 'TEXTAREA'
+            || document.activeElement.isContentEditable) {
+          return false;
+        }
 
-          if (keyCode == tc.settings.rewindKeyCode) {
-            runAction('rewind', document, true)
-          } else if (keyCode == tc.settings.advanceKeyCode) {
-            runAction('advance', document, true)
-          } else if (keyCode == tc.settings.fasterKeyCode) {
-            runAction('faster', document, true)
-          } else if (keyCode == tc.settings.slowerKeyCode) {
-            runAction('slower', document, true)
-          } else if (keyCode == tc.settings.resetKeyCode) {
-            runAction('reset', document, true)
-          } else if (keyCode == tc.settings.displayKeyCode) {
-            runAction('display', document, true)
-          } else if (keyCode == tc.settings.fastKeyCode) {
-            runAction('fast', document, true);
+        if (keyCode == tc.settings.displayKeyCode) {
+          runAction('display', document, true)
+        }
+        var item = tc.settings.keyBindings.find(item => item.key === keyCode);
+        if (item) {
+          runAction(item.action, document, item.value);
+          if (item.force === "true") {// disable websites key bindings
```

Comment:
On my tests it's always first one but it's disabled in settings page for now. Because it also stop everything like space to page down even though there is no video